Laravel-Backpack/CRUD
# Select2 focus as select

## WHY

### BEFORE - What was wrong? What was happening before this PR?

A lot of normal behavior was broken because auto-open select2.

https://github.com/Laravel-Backpack/PRO/pull/81
https://github.com/Laravel-Backpack/CRUD/issues/4606
https://github.com/Laravel-Backpack/CRUD/issues/4605

### AFTER - What is happening after this PR?

Auto-open select2 was update to auto-focus, and add style to emulate normal field halo border.


## HOW

### How did you achieve that, in technical terms?

`firstField.trigger('open');` was updated to `firstField.trigger('focus');`


### Is it a breaking change?

Yes, because early update broke many normal fields attribute like "min_rows" as example


### How can we test the before & after?

First test, just focus any select2 need to be able to press "space" key and open list, up and down to navigate.

Second, check all this bugs

https://github.com/Laravel-Backpack/PRO/pull/81
https://github.com/Laravel-Backpack/CRUD/issues/4606
https://github.com/Laravel-Backpack/CRUD/issues/4605
